test/test_core.py

```diff
@@ -5866,7 +5866,10 @@ def test_fs_readdir_ino_matches_stat_ino(self):
     self.do_runf('fs/test_fs_readdir_ino_matches_stat_ino.c', 'success')
 
   @also_with_nodefs_both
+  @crossplatform
   def test_fs_open_no_permissions(self):
+    if '-DNODEFS' in self.emcc_args or '-DNODERAWFS' in self.emcc_args and WINDOWS:
```

Comment:
Does this need brackets?  I can't remember of hand is `and` or `or` has a higher precedence in python.